Core/Systems/KeywordSystem/KeywordScanner.cs

```diff
@@ -27,6 +27,7 @@ internal class KeywordScanner : GlobalItem
 	{
 		public static List<Keyword> keywords = new();
 
+		[CloneByReference]
```

Comment:
Seems like this should be `static` instead

Reply:
Nah this is the instanced subset of keywords a given item is holding